protos/sigstore_trustroot.proto

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.trustroot.v1;
+
+import "sigstore_common.proto";
+
+option go_package = "github.com/sigstore/proto/trustroot/v1";
+option java_package = "dev.sigstore.proto.trustroot.v1";
+option java_multiple_files = true;
+option java_outer_classname = "TrustRootProto";
+
+// TransparencyLogInstance describes the immutable parameters from a
+// transparency log.
+// See https://www.rfc-editor.org/rfc/rfc9162.html#name-log-parameters
+// for more details.
+// The incluced parameters are the minimal set required to identify a log,
+// and verify an inclusion promise.
+message TransparencyLogInstance {
+        // The base URL at which can be used to URLs for the client.
+        string base_url = 1;
+        // The hash algorithm used for the Merkle Tree.
+        dev.sigstore.common.v1.HashAlgorithm hash_algorithm = 2;
+        // The signature algorithm used.
+        dev.sigstore.common.v1.SignatureAlgorithm signature_algorithm = 3;
```

Comment:
Isn't this now redundant with `public_key.KeyFormat`?

Reply:
It is, let's drop it.